cosmos-auth/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cosmos-auth",
   "description": "Node.js app exposing Cosmos Auth API",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
As in https://github.com/telefonicaid/perseo-fe/blob/develop/package.json#L4 (another Node.js project using that approach)